### PR TITLE
added vtx_smartaudio_early_akk_workaround option

### PR DESCRIPTION
**Pull request by RomanLut**
Solution for https://github.com/iNavFlight/inav/issues/6456

### CODE REVIEW

**Comment by RomanLut** [comment]
I would like to ask someone to run  src/utils/update_cli_docs.py because I have problems running it on my dev env.

**Comment by DzikuVx** [comment]
@RomanLut please update docs with `python3 src/utils/update_cli_docs.py`

**Comment by RomanLut** [comment]
Unfortunately I have problem running this script under MacOS:

Traceback (most recent call last):
  File "src/utils/update_cli_docs.py", line 55, in <module>
    md_table_lines = generate_md_table_from_yaml(settings_yaml)
  File "src/utils/update_cli_docs.py", line 37, in generate_md_table_from_yaml
    md_table_lines.append("| {} | {} | {} |\n".format(param[0], param[1]['default'], param[1]['description']))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb0' in position 62: ordinal not in range(128)

There is indeed degree symbol u'\xb0' in the settings.yaml.

I would kindly ask someone runt this script or help me.

**Comment by DzikuVx** [comment]
@RomanLut please try now after merging `master` to your local branch.

**Comment by RomanLut** [comment]
1) Updated local master from master
2) installed pyyaml
3) run script
now I get a lot of errors:
Mac-Admin:inav-master admin$ python3 src/utils/update_cli_docs.py
Setting "gyro_notch_hz" has no description or default value specified
Setting "gyro_notch_cutoff" has no description or default value specified
Setting "gyro_to_use" has no description or default value specified
Setting "acc_notch_hz" has no description or default value specified
Setting "acc_notch_cutoff" has no description or default value specified
Setting "opflow_scale" has no description or default value specified

no files are changed after running script.

**Comment by RomanLut** [comment]
Probably I have to open separate issue about running update_cli_docs.py on macOs.
For now, I updated settings.md manually.
I hope it will work.

File: src/main/io/vtx_control.h [hunk 1]
```diff
@@ -31,6 +31,7 @@ typedef struct vtxChannelActivationCondition_s {
 typedef struct vtxConfig_s {
     vtxChannelActivationCondition_t vtxChannelActivationConditions[MAX_CHANNEL_ACTIVATION_CONDITION_COUNT];
     uint8_t halfDuplex;
+    uint8_t smartAudioEarlyAkkWorkaroundEnable;
```
Comment (DzikuVx) [inline src/main/io/vtx_control.h:34]: @RomanLut we are missing the default initialization of new setting in the VTX parameter group. This causes initialization to OFF. 
Also, PG version bump is missing

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/io/vtx_control.h (hunk 1 above)
Comment (RomanLut) [reply src/main/io/vtx_control.h:34]: 1) Do you think default should be ON or OFF?
From one side, OFF will change existing functionality, but from other side, this is workaround, so it should be OFF by default. As I mentioned, new AKK AIO camera does not work with workaround enabled.
2) What do you mean with "PG version bump is missing" ?

Comment (DzikuVx) [reply src/main/io/vtx_control.h:34]: 1. Should be ON by default as compatible with previous versions
2. See here https://github.com/iNavFlight/inav/pull/6477/files#diff-a680ca1e23901579e9214c055509004e8b5c05e3f7281ed74ac1edfc38caabaeR188 as an example. `smartAudioEarlyAkkWorkaroundEnable` should be initialized in `PG_RESET_TEMPLATE` for the VTX config and version should be increased in `PG_REGISTER_WITH_RESET_TEMPLATE`.  In here https://github.com/iNavFlight/inav/blob/master/src/main/io/vtx_control.c#L42

Comment (RomanLut) [reply src/main/io/vtx_control.h:34]: Added initialization to ON, increased version.

**Review by DzikuVx — APPROVED** [review]

**Comment by DzikuVx** [comment]
Thanks a lot, merging!

**Pull request merged**